keyboards/keebio/nyquist/keymaps/skug/keymap.c

```diff
@@ -24,9 +24,10 @@ extern keymap_config_t keymap_config;
 // Layer names don't all need to be of the same length, obviously, and you can also skip them
 // entirely and just use numbers.
 #define BASE 0
-#define SYMB 1
-#define MDIA 2
-#define ARRW 3
+#define GAME 1
+#define SYMB 2
+#define MDIA 3
+#define ARRW 4
 #define _ADJUST 16
```

Comment:
I agree with @zvecr; I'd very much like to see an enum used here. If you replace Lines 26-31 with the following code, it'll fix this right up. Everything else on your PR looks good.

```c
enum layer_names {
    BASE,
    GAME,
    SYMB,
    MDIA,
    ARRW,
    _ADJUST,
};
```

This won't change the behavior of your keymap at all.

Reply:
switched to an enum as requested